src/commands/load.js

```diff
@@ -13,8 +19,32 @@ export function load(line) {
     if (!fs.existsSync(filePath)) {
       throw Error(chalk.red('ERROR: file not found.'));
     }
-    console.log(chalk.green('Loading... ', fn));
-    line = fs.readFileSync(filePath, 'utf-8');
+    console.log(chalk.green('Loading... ', fn))
+    line = fs.readFileSync(filePath, 'utf-8')
+
+    const spinner = ora({
+      spinner: 'dots',
+      suffixText: ``
+    })
+    spinner.start()
+    spinner.suffixText = chalk.gray('Parsing project structure...')
+    const projectStructure = createProjectStructure(
+      line,
+      path.dirname(filePath)
+    )
+    if (projectStructure.length > 0) {
+      line = createExecutableFromProject(projectStructure) + '\n\n' + line
+    }
+    spinner.stop()
+
+    if (projectStructure.length > 0) {
+      console.log(chalk.yellow('\nThe following files will be deployed:'))
+      console.log(chalk.dim(createFileTree([
```

Comment:
Since we are printing the project structure, it might make sense to ask the user after this if they're sure about loading the project. Larger workspaces could include sensitive files and the file tree could be a warning to them about these files.

Reply:
Agree 💯

Reply:
May want to ignore foo.test.lua files too. 🤔

Reply:
I don't believe there can be a file like `foo.test.lua`, because it would break Lua requires.
`foo.test.lua` would be loded from the path `/foo/test.lua`.

Reply:
(also if those files are not dependencies of the main file, they won't be loaded)